### PR TITLE
OpenJ9 user documentation .css for latest dependency levels Fix

### PR DESCRIPTION
**Pull request by Bazif-Khan**
Signed-off-by: Bazif Khan <bazif.khan@ibm.com>

### CODE REVIEW

**Comment by pmhayward** [comment]
Just for the record, the latest levels that we need to update to and which these changes work with are:
mkdocs   1.0.4
material   4.4.3

**Comment by SueChaplain** [comment]
Full dependency list I just generated by asking just for `mkdocs` and `mkdocs-material`:

click==7.0                # via mkdocs
htmlmin==0.1.12           # via mkdocs-minify-plugin
jinja2==2.10.3            # via mkdocs
jsmin==2.2.2              # via mkdocs-minify-plugin
livereload==2.6.1         # via mkdocs
markdown==3.1.1           # via mkdocs, mkdocs-material, pymdown-extensions
markupsafe==1.1.1         # via jinja2
mkdocs-material==4.5.0
mkdocs-minify-plugin==0.2.1  # via mkdocs-material
mkdocs==1.0.4
pep562==1.0               # via pymdown-extensions
pygments==2.4.2           # via mkdocs-material
pymdown-extensions==6.2   # via mkdocs-material
pyyaml==5.1.2             # via mkdocs
six==1.13.0               # via livereload
tornado==6.0.3            # via livereload, mkdocs


Note: Seems that pip-tools will not overwrite a requirements.txt file if one exists (seem to remember this now!), which explains why I was not picking up the later versions when running this last week in a container. Currently we run `pip-compile` when the docker image is generated, so `/tmp/requirements.txt` shows the latest levels that were available when the image was created (stored in a layer). `/tmp/requirements.orig` shows the levels that we install for the container.

Interestingly:

- I removed pygments from the .in list, and mkdocs-material puts it back!
- material has moved up a notch since last week :-/

**Comment by pmhayward** [comment]
These are the changes to the hash codes in `base.html` between version 4.4.3 and 4.5.0:

| Line # | 4.4.3                             | 4.5.0                             |
|--------|-----------------------------------|-----------------------------------|
| 74     |`application.30686662.css`         |`application.1b62728e.css`         |
| 76     |`application-palette.a8b3c06d.css` |`application-palette.a8b3c06d.css` |
| 87     |`modernizr.74668098.js`            |`modernizr.268332fc.js`            |
| 218    |`application.ac79c3b0.js`          |`application.a8b5e56f.js`          |

(Line 76  doesn't seem to have changed?)

File: theme/base.html [hunk 1]
```diff
@@ -71,9 +71,9 @@
       {% endif %}
     {% endblock %}
     {% block styles %}
-      <link rel="stylesheet" href="{{ 'assets/stylesheets/application.451f80e5.css' | url }}">
+      <link rel="stylesheet" href="{{ 'assets/stylesheets/application.30686662.css' | url }}">
```
Comment (pmhayward) [inline theme/base.html]: For version 4.5.0, please update to: `application.1b62728e.css`

File: theme/base.html [hunk 2]
```diff
@@ -71,9 +71,9 @@
...
       {% if palette.primary or palette.accent %}
-        <link rel="stylesheet" href="{{ 'assets/stylesheets/application-palette.22915126.css' | url }}">
+        <link rel="stylesheet" href="{{ 'assets/stylesheets/application-palette.a8b3c06d.css' | url }}">
```
Comment (pmhayward) [inline theme/base.html]: For version 4.5.0, this reference seems to have stayed the same: `application-palette.a8b3c06d.css`, so no update needed.

File: theme/base.html [hunk 3]
```diff
@@ -84,7 +84,7 @@
       {% endif %}
     {% endblock %}
     {% block libs %}
-      <script src="{{ 'assets/javascripts/modernizr.1aa3b519.js' | url }}"></script>
+      <script src="{{ 'assets/javascripts/modernizr.74668098.js' | url }}"></script>
```
Comment (pmhayward) [inline theme/base.html]: For version 4.5.0, please update to: `modernizr.268332fc.js`

File: theme/base.html [hunk 4]
```diff
@@ -215,7 +215,7 @@ <h2 id="__source">{{ lang.t("meta.source") }}</h2>
       {% endblock %}
     </div>
     {% block scripts %}
-      <script src="{{ 'assets/javascripts/application.583bbe55.js' | url }}"></script>
+      <script src="{{ 'assets/javascripts/application.ac79c3b0.js' | url }}"></script>
```
Comment (pmhayward) [inline theme/base.html]: For version 4.5.0, please update to: `application.a8b5e56f.js`

**Comment by pmhayward** [comment]
@Bazif-Khan ...  One more tweak to oj9.css, please. I don't think you halved the rem value on line 146 ... `.md-typeset h3 code {font-size: 1.9rem;` Needs to be `0.95rem`

**Pull request merged**